Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN cargo build --release
 
 FROM debian:buster-slim
 
-RUN apt-get update && apt-get install -y libssl-dev ca-certificates
+RUN apt-get update && apt-get install -y libssl-dev ca-certificates sqlite3
```

Comment:
@lamafab please note that I left the sqlite3 here, being the fact you will need it for your next release... do you think it may cause somehow an issue/conflict with the current version ?

Reply:
No, this is good.